sound/soc/codecs/rt700-sdw.c

```diff
@@ -496,6 +496,7 @@ static int rt700_dev_suspend(struct device *dev)
 		return 0;
 
 	regcache_cache_only(rt700->regmap, true);
+	regcache_mark_dirty(rt700->regmap);
```

Comment:
it's not bad, but when we implement the clock stop mode where the Intel DSP remains powered (D0i3), then forcing the cache to be resync'ed will be unnecessary. It'd be better to only did this when we actually to the io_init is done (after the bus reset)

Reply:
OK, we let the regcache_mark_dirty API move to the end of io_init function like below.
```suggestion
	if (rt711->first_init) {
		regcache_cache_bypass(rt711->regmap, false);
+		regcache_mark_dirty(rt711->regmap);
	} else
```
We may change the first_init name to hw_init_reset? Is it suitable?

Reply:
maybe hw_init and first_hw_init ?

that'd explain what we done only once, and what we do for all cases.

Reply:
OK, I will submit a new PR for it today.